packages/patternfly-4/react-charts/src/components/ChartLabel/ChartLabel.tsx

```diff
@@ -166,8 +166,20 @@ export interface ChartLabelProps extends VictoryLabelProps {
   y?: number;
 };
 
-export const ChartLabel: React.FunctionComponent<ChartLabelProps> = (props: ChartLabelProps) =>
-  <VictoryLabel {...props} />;
+export const ChartLabel: React.FunctionComponent<ChartLabelProps> = ({
+  style,
+  ...rest
+}: ChartLabelProps) => {
+  const newStyles: React.CSSProperties[] = [];
+  if (Array.isArray(style)) {
+    for(let i = 0; i < style.length; i++) {
+      newStyles.push(defaults({}, style[i], LabelStyles));
+    }
+  } else {
+    newStyles.push(style ? defaults({}, style, LabelStyles) : LabelStyles);
+  }
+  return <VictoryLabel style={newStyles as any} {...rest} />;
+}
```

Comment:
This could be simplified:
```js
  const applyDefaultStyle = (customStyle) => defaults(customStyle, LabelStyles);
  const newStyle = Array.isArray(style) ? style.map(applyDefaultStyle) : applyDefaultStyle(style);
  return <VictoryLabel style={newStyle as any} {...rest} />; 
```

Reply:
Updated, thanks